etc/systemd/dnf-automatic-install.timer

```diff
@@ -2,6 +2,8 @@
 Description=dnf-automatic-install timer
 # See comment in dnf-makecache.service
 ConditionPathExists=!/run/ostree-booted
+Wants=network-online.target
```

Comment:
I'm not sure if automatic-install should require network because it is supposed to install packages which are already downloaded.